packages/components/src/Layout/Box/Box.test.tsx

```diff
@@ -25,146 +25,57 @@
-    assertSnapshot(<Box mt={['large', 'medium', 'small']}>🥑</Box>)
-  })
-
-  test('Box with null values are removed from styling', () => {
-    assertSnapshot(<Box mt={['large', null, 'medium']}>🥑</Box>)
-  })
-
-  test('Box with SizeNone is valid', () => {
-    assertSnapshot(<Box mt="none">🥑</Box>)
-  })
-
-  test('Box supports background-color', () => {
-    assertSnapshot(<Box bg="key" />)
-  })
-
-  test('Box supports display', () => {
-    assertSnapshot(<Box display="inline-block" />)
-  })
-
-  test('Box supports height properties', () => {
-    assertSnapshot(<Box height="1vh" />)
-    assertSnapshot(<Box maxHeight="1vh" />)
-    assertSnapshot(<Box minHeight="1vh" />)
-  })
-
-  test('Box supports width', () => {
-    assertSnapshot(<Box width="1vw" />)
-    assertSnapshot(<Box maxWidth="1vw" />)
-    assertSnapshot(<Box minWidth="1vw" />)
-  })
-
-  test('Box supports position, top, left, bottom, right', () => {
-    assertSnapshot(<Box position="absolute" />)
-    assertSnapshot(<Box top="1rem" />)
-    assertSnapshot(<Box left="1rem" />)
-    assertSnapshot(<Box right="1rem" />)
-    assertSnapshot(<Box bottom="1rem" />)
-  })
-
-  test('Box supports responsive fontSize and lineHeight', () => {
-    assertSnapshot(<Box fontSize={['small', 'large']} />)
-    assertSnapshot(<Box lineHeight={['small', 'large']} />)
-  })
-
-  describe('borders', () => {
-    test('Box supports borders', () => {
-      assertSnapshot(<Box border="1px solid black" />)
-    })
-
-    test('supports borderRadius', () => {
-      assertSnapshot(<Box borderRadius="medium" />)
-    })
-
-    test('supports borderColor', () => {
-      assertSnapshot(<Box borderColor="ui2" />)
-    })
-  })
-
-  describe('fonts', () => {
-    test('font helpers', () => {
-      assertSnapshot(<Box fontFamily="brand" />)
-      assertSnapshot(<Box fontSize="small" />)
-      assertSnapshot(<Box fontWeight="bold" />)
-      assertSnapshot(<Box letterSpacing="10" />)
-      assertSnapshot(<Box lineHeight="medium" />)
-      assertSnapshot(<Box textAlign="right" />)
-      assertSnapshot(<Box color="ui4" />)
-    })
-  })
-
-  describe('flex', () => {
-    test('supports flex properties', () => {
-      assertSnapshot(
-        <Box
-          display="flex"
-          alignContent="flex-end"
-          alignItems="baseline"
-          justifyContent="center"
-          flexWrap="wrap"
-          flexDirection="column"
-        />
-      )
-    })
-
-    test('supports flex item properties', () => {
-      assertSnapshot(
-        <Box alignSelf="center" flexBasis="1" flex="1 1" order="unset" />
-      )
-    })
-  })
-
   describe('as=', () => {
     test('allows Box to render as any HTML tag', () => {
-      const boxAsButton = mountWithTheme(<Box as="button" />)
-      expect(boxAsButton.find('button').exists()).toBeTruthy()
+      renderWithTheme(<Box as="button">Press Me</Box>)
+      expect(screen.getByText('Press Me').tagName).toEqual('BUTTON')
```

Comment:
Could you use `getByRole` here?

```suggestion
      expect(screen.getByRole('button')).toBeInTheDocument()
```

Reply:
oh yep, that's better! good call